build/org.eclipse.pde.build/src/org/eclipse/pde/internal/build/Utils.java

```diff
@@ -934,21 +874,11 @@ static public int scan(StringBuffer buf, int start, String[] targets) {
 	 * @throws IOException
 	 */
 	static public StringBuffer readFile(File target) throws IOException {
-		return readFile(new FileInputStream(target));
+		return new StringBuffer(new String(Files.readAllBytes(target.toPath())));
```

Comment:
Files.readString uses UTF8 instead of std charset. Probably UTF8 is meant anyway, but this change is not about changing the charset.

Reply:
That's right that the charset does not match, but then the default charset can be passed as second argument.
This would have to advantages, it saves one string-array copy (`Files.readString()` re-uses the byte array in the string) and it makes more clear that a 'special' charset is used here and future readers of the code are less likely to over-see it.
Furthermore a second argument is not that much more code and then we can replace this extra method with standard Java.